cfgov/v1/migrations/0006_deprecate_raft.py

```diff
@@ -0,0 +1,29 @@
+        ),
+        migrations.AlterField(
+            model_name='cfgovcontentowner',
+            name='slug',
+            field=models.SlugField(max_length=100, unique=True, verbose_name='slug'),
+        ),
```

Comment:
```suggestion
```

I'm puzzled why this part was generated. Do you possibly have an older or newer version of Django or Wagtail installed locally? Deleting these lines _should_ resolve the validate-migrations failure.

Reply:
I'm not sure. I committed this change, we'll see what happens